lib/sprockets/asset.rb

```diff
@@ -123,6 +125,11 @@ def digest
       metadata[:digest]
     end
 
+    # Public: Returns String version of source.
```

Comment:
This is not the version of the source but of the environment, so I think we should make that explicit and document as:

```
Private: Return the version of the environment were the asset was generated.